src/signals/incident/components/form/DescriptionInput/DescriptionInput.js

```diff
@@ -0,0 +1,49 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import TextArea from 'components/TextArea';
+import DescriptionInfo from '../DescriptionInfo';
+
+function get(e, meta, parent) {
```

Comment:
Can you rename `e` to `event` for readability?

Reply:
Sure!